src/appReview/saga.ts

```diff
@@ -0,0 +1,18 @@
+import InAppReview from 'react-native-in-app-review'
+import { call, takeLatest } from 'redux-saga/effects'
+import { Actions, initializeAppReview, updateAppReview } from 'src/appReview/actions'
+import { safely } from 'src/utils/safely'
+
+export function* initAppReview() {
+  yield call(initializeAppReview)
+}
+
+export function* setAppReview() {
+  const result = yield call(InAppReview.RequestInAppReview)
+  yield call(updateAppReview, result)
+}
+
+export function* appReviewSaga() {
+  yield takeLatest(Actions.INITIALIZE_APP_REVIEW, safely(initAppReview))
```

Comment:
i'm lost here. what do you want to do? it seems like you are taking the latest INITIALIZE_APP_REVIEW action, triggering the initAppReview saga which just dispatches the INITIALIZE_APP_REVIEW again? am i reading this right?

(by the way, we should use `call` to call functions, if you want to dispatch an action then we should use `put` instead)

Reply:
I've adjusted the PR a bit since this last review. Instead we are now listening to the action `Web3Actions.SET_ACCOUNT` for `initAppReview` and `SendActions.SEND_PAYMENT_SUCCESS` for `setAppReview`.